src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -132,13 +133,14 @@
     <SqlScript Include="Features\Schema\Sql\Scripts\FullText.sql" />
   </ItemGroup>
   
+  <!-- Properties used by sql task to generate full script-->
   <PropertyGroup>
     <LatestSchemaVersion>5</LatestSchemaVersion>
     <IntermediateOutputPath>$(MSBuildThisFileDirectory)obj\$(Configuration)</IntermediateOutputPath>
```

Comment:
Is there a configuration option to leave formatting alone? The current task is stripping out comments and reducing whitespace (for example, consolidating the parameters onto one line) in stored procedures. This is causing the schema comparison test to fail.

Reply:
Yup, the diff of Prc is literal Bodyscript compare. I am skipping that compare for now. We will still